src/index/script/HeadsUpDisplay/StatusWidget/_StatusWidget.coffee

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Globals from '../../../../_common/script/Globals.coffee'
+
+export default class StatusWidget extends React.Component
+
+	############################################################################
+
+	constructor: (props) ->
+		super props
+
+	############################################################################
+
+	render: =>
+		<form onSubmit={this.handleSubmit}>
+			<button type="submit" className="widget" id="status-widget">
+				{this.props.status}
+			</button>
+		</form>
+
+	#///////////////////////////////////////////////////////////////////////////
```

Comment:
These banner comments were the result of me experimenting with a style change.

I noticed while reading _Clean Code_ that it was difficult to tell at a glance how important to the program's logic any one particular function was. It was also difficult to easily tell when I had reached the end of the implementation for any one high-level function and moved on to the next, because all functions were marked the same way and treated with the same importance. I attempted to modify my existing banner comments to better distinguish high-level functions from low-level functions.

A good example of this is my work-in-progress websocket manager microservice. Here's a link to the `ClientConnection` class from that project: https://pastebin.com/aS1vh2Fb

The logic flow when calling `NewClientConnection` looks like this:

```
NewClientConnection
    instantiateEmptyClientConnection
        populateUserData
            populateSessionId
            populateUsername
            populateUserRole
    establishWSConnection
        configureWSUpgrader
            checkWSOrigin
        performUpgrade
        configureWSConnection
            setReadLimitByRole
    beginHandlingMessages
        handleSendMessages
            sendMessage
        handleReceiveMessages
            receiveMessage
        closeWSConnection
    sendInitializationMessages
```

I introduced banner comments between these different functions to visually separate them. There's a thick banner between `setReadLimitByRole` and `beginHandlingMessages` to show that the two functions aren't really related, but there's a very thin banner between `setReadLimitByRole` and `configureWSConnection` because those two are closely related -- the latter directly calls the former.

In the file you commented on here, the hierarchy looks like this:

```
constructor
render
    handleSubmit
```

The constructor doesn't really call anything, but `render` calls `handleSubmit`, so there's a less aggressive banner between the two of them to indicate less separation.

These banner comments are (in my opinion) helpful in a large file, like the one I linked above, but they're not really that helpful in such a small stub. I'll get rid of them from this file.